instrumentation/opentelemetry_req/lib/opentelemetry_req.ex

```diff
@@ -106,12 +122,21 @@ defmodule OpentelemetryReq do
   defp format_exception(_), do: ""
 
   defp span_name(request) do
-    Req.Request.get_private(request, :path_params_template)
+    case request.options[:span_name] do
+      nil ->
+        route = Req.Request.get_private(request, :path_params_template) || request.url.path
```

Comment:
This should not use the url path since that is a high cardinality value. The spec says for clients to just use method as the span name.

Reply:
Oh right, just re-read the spec. Thanks! Addressed